code/index.html

```diff
@@ -3,20 +3,127 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-1/css/all.min.css"
+      integrity="sha512-ZYg+hQvLlM0W9C3P6vTpes5LmJ66G2EsroxDNzwD6qiulckldb29eD9I672HL2X+LH5nRhY1sQqZLJGB+aUUPg=="
+      crossorigin="anonymous"
+    />
+
+    <!--Allow us to use icons-->
+
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css2?family=Newsreader&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Hina+Mincho&display=swap"
+      rel="stylesheet"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
i think adding comments before the actual code will increase the readability and understanding

Reply:
Sure!! I did not have enough time to review my code one more time, I spent so much time on flexbox that I forgot the most basic.